### DISCUSSION TITLE
feat: Highlight Active Volunteers on the Member Page

### DISCUSSION THREAD

**Discussion by adiati98** [Virtual-Coffee/virtualcoffee.io#1364] [VC Tech]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Issue Context

Currently, active volunteers have badge of their roles on the member page. However, having a section dedicated for them on the member page would be great for these reasons:

- A gesture of appreciation to their valuable support
- Other VC members can easily identify active volunteers
- It would be easier to include the link to this section on resume or social media for active volunteer who are looking for jobs or looking for other opportunities.

### Proposed solution

- Create an "Active Volunteers" section right under "Core Team".
- Move active volunteers to this section.
- Provide clickable link to each sections (Core Team, Active Volunteers, and Our Members).

### Code of Conduct

- [X] I've read the Code of Conduct and understand my responsibilities as a member of the Virtual Coffee community

**Comment by eternalmaha** [comment]
I see some people have indicators like “Monthly Team Lead” or “Notetakers” as shown by a gray box and white lettering, and then theres others that have smaller badgers under their name that say things like “Hacktoberfest 2024 contributer”. Do I include both categories of people or just the ones with the gray boxes? @adiati98 @meg-gutshall

**Comment by adiati98** [comment]
Hey @eternalmaha, we will include the ones with the gray labels. The "Hacktoberfest XXXX contributor" is badge for folks who participated in Hactoberfest.

Currently, we don't have indication in active and past volunteers. So folks that have labels are either still active or not anymore. We might want to consider having both active and past volunteers. We can later on give you the list of active (and past) volunteers. 👍🏼 

I have a couple of thoughts to indicate active and past volunteers:

- Give green label for active ones and red (or yellow) for past volunteers. 
- Have two sections of "Active Volunteers" and "Past Volunteers".

Would love to have @Virtual-Coffee/maintainers thoughts on this.

**Comment by eternalmaha** [comment]
@adiati98 I think highlighting active volunteers in a separate section can be really helpful while also including a green badge next to their name in the full member list. For past inactive volunteers, if we choose to include them, my idea is to have an expandable section user would need to click under current active volunteers that then reveals the past ones, so the focus is still on the active ones so its more accessibility and clean?

**Comment by adiati98** [comment]
@eternalmaha I think that could be one option.

Another option is to have a dedicated page for volunteers that will have sections of active and inactive volunteers to prevent the members page being too long. But we might still want to have the labels on the member's profile.

I'm thinking to have a list of the volunteer names and roles. Ideally, we can click on the volunteer's name that can open their profile on members' page.

Here are some examples of community's team pages for ideas:

- [The Collab Lab](https://the-collab-lab.codes/mentors/)
- [Astro](https://github.com/withastro/.github/blob/main/MEMBERS.md) -- although they only have the team list on GH, we can picture if we have this on website.
- [The Good Docs Project](https://www.thegooddocsproject.dev/our-team)
- Mautic community -- they have two pages:
  - [Community Handbook](https://contribute.mautic.org/community-leadership/current-leadership-team)
  - [Website](https://mautic.org/leadership/)

---

@meg-gutshall can you help with the list of active & inactive volunteers? Mine is not complete. 😅 

@Virtual-Coffee/maintainers do you have thoughts and ideas on this?

**Comment by BekahHW** [comment]
I like @adiati98's proposed solution, but it might be worth having an alumni volunteer section below the active volunteers bc we're still grateful and it's still an opportunity for them to put it on their resume.

**Comment by meg-gutshall** [comment]
I need to schedule some time in my calendar to look into this further.

I'm going to convert this to a discussion and post the link in our Slack to get suggestions from the community. We have all these creative minds at our disposal, why not use them?

@eternalmaha, we will keep you assigned to this issue. Once we have a final design, you may decide you want to pair with another contributor (or two) on this issue. If so, let us know and we can assign them! 👍
